garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua

```diff
@@ -213,7 +213,7 @@ L.sb_mia = "Eylem Eksik"
 L.sb_confirmed = "Onaylanmış Ölü"
 
 L.sb_ping = "Gecikme"
-L.sb_deaths = "Ölümler"
+L.sb_deaths = "Ölüm"
```

Comment:
I don't think this change is correct? The English string for this is plural, not singular.

https://github.com/Facepunch/garrysmod/blob/33740f86eea326dbc7f84104e993f7c769c65329/garrysmod/gamemodes/terrortown/gamemode/lang/english.lua#L216

Reply:
Its better this way. The change mimics the CS2 scoreboard and its generally used singular if its in a scoreboard.

Reply:
![image](https://github.com/user-attachments/assets/5462749e-ea37-411e-9e41-ba9f1c98a8c1)
Here is an example.